HISTORY.md

```diff
@@ -1,10 +1,47 @@
+## v1.10.9 (2023-06-07)
```

Comment:
can you push `1.10.X` before the V2 releases.

Reply:
done